cheesecakecore/src/main/java/io/cheesecakelabs/cheesecakecore/utils/extensions/View+Visibility.kt

```diff
@@ -0,0 +1,21 @@
+package io.cheesecakelabs.cheesecakecore.utils.extensions
+
+import android.view.View
+
+fun View.visible() {
+    this.visibility = View.VISIBLE
+}
+
+fun View.gone() {
+    this.visibility = View.GONE
+}
+
+fun View.invisible() {
+    this.visibility = View.INVISIBLE
+}
+
+var View.isVisible: Boolean
+    get() = this.visibility == View.VISIBLE
+    set(value) {
+        this.visibility = if (value) View.VISIBLE else View.GONE
```

Comment:
I think it is not clear what will happen if I set it to false, if it will set to INVISIBLE or GONE. Maybe this should just be a function that returns true if it is VISIBLE

Reply:
That's true, @gianebinotto asked about this as well. I used it this way in tapcart because I almost never want to make the view invisible.

Reply:
Solved